src/network_generator.py

```
@@ -122,7 +121,7 @@ def visualize_network(
             return True
 
         except Exception as e:
-            logger.error(f"Error occurred while visualizing network: {e}", exc_info=True)
+            logger.exception(f"Error occurred while visualizing network: {e}")
```

Ruff (0.14.2)

Redundant exception object included in `logging.exception` call

(TRY401)